include/emitc/emitc_mhlo.h

```diff
@@ -328,6 +328,7 @@ inline Src logical_xor(Src x, Src y) {
 
 /// Functions for other MHLO ops.
 // BroadcastInDimOp
+// Maps the broadcast_dimensions from Src to Dest
```

Comment:
```suggestion
// Maps the broadcast dimensions from Src to Dest
```
?

Reply:
Oh, even better with your latest commit :)

Reply:
Just updated the comment as you were typing your review! :)